src/pipecat/services/elevenlabs.py

```diff
@@ -167,34 +180,6 @@ async def set_model(self, model: str):
         await self._disconnect()
         await self._connect()
 
-    async def set_voice(self, voice: str):
```

Comment:
same here. this has custom handling that's more than just setting the voice.... updating my theoretical code below to include a potential way to handle it.

Reply:
nevermind. that one can be ignored... i think this needs to be added to custom handle voice changes though:

```
async def _update_settings(self, settings: Dict[str, Any]):
  prev_voice = self._settings["voice"]
  super()._update_settings(settings)
  if not prev_voice == self._settings["voice"]:
    await self._disconnect()
    await self._connect()
```

TBH. The fact that this behavior also happens when the model changes, I'm wondering if elevenlabs requires a disconnect/connect to apply ANY changes 🤔